core/scripts/ocr2vrf/util.go

```diff
@@ -311,3 +312,11 @@ func decodeHexTo32ByteArray(val string) (byteArray [32]byte) {
 	copy(byteArray[:], decoded)
 	return
 }
+
+func deployLinkEthFeed(e helpers.Environment, linkAddress string, weiPerUnitLink *big.Int) common.Address {
+	_, tx, _, err :=
+		mock_v3_aggregator_contract.DeployMockV3AggregatorContract(
+			e.Owner, e.Ec, 18, weiPerUnitLink)
+	helpers.PanicErr(err)
+	return helpers.ConfirmContractDeployed(context.Background(), e.Ec, tx, e.ChainID)
+}
```

Comment:
noice

Reply:
Removed, using the one in helpers instead.